packages/core/src/App/Containers/ResetEmailModal/confirm-email-modal.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { SentEmailModal } from '@deriv/account';
+import { WS } from 'Services';
+import PropTypes from 'prop-types';
+import { connect } from 'Stores/connect';
+
+export const ConfirmEmailModal = ({ onClose, is_open, prevEmail, newEmail, verification_code, pullErrorMessage }) => {
+    const [email_request, setEmailRequest] = React.useState(null);
+    const [is_send_email_modal_open, setIsSendEmailModalOpen] = React.useState(false);
+    const [is_error, set_is_error] = React.useState(false);
+    const [error_message, set_error_message] = React.useState(null);
+    const [isOpen, setIsOpen] = React.useState(is_open);
+
+    const handleSubmit = () => {
+        const api_request = {
+            change_email: 'verify',
+            new_email: newEmail,
+            verification_code,
+        };
+
+        setEmailRequest(prev => ({ ...prev, ...api_request }));
+
+        WS.changeEmail(api_request).then(response => {
+            if (response.error) {
+                setIsOpen(false);
+                onClose(true);
+                set_is_error(true);
+                set_error_message(response.error.message);
+                pullErrorMessage(response.error.message);
+            } else {
+                setIsSendEmailModalOpen(true);
+                setIsOpen(false);
+            }
+        });
+    };
+
+    const resendEmail = () => {
+        WS.changeEmail(email_request);
+    };
+
+    if (is_send_email_modal_open) {
+        return (
+            <SentEmailModal
+                is_open={is_send_email_modal_open}
+                onClose={() => setIsSendEmailModalOpen(false)}
+                identifier_title={'Change_Email'}
+                onClickSendEmail={resendEmail}
+                has_live_chat={true}
+                is_modal_when_mobile={true}
+            />
+        );
+    }
+    return (
+        <Modal
+            is_open={isOpen}
+            should_header_stick_body
+            title={<Localize i18n_default_text='Are you sure?' />}
+            toggleModal={onClose}
+            width='440px'
+        >
+            {is_error ? <p>{error_message}</p> : null}
+            <React.Fragment>
+                <div className='email-confirmation'>
+                    <Text as='p' color='prominent' size='xs' align='left'>
+                        <Localize
+                            i18n_default_text='Are you sure you want to update email <0>{{prevEmail}}</0> to <1>{{newEmail}}</1>?'
+                            values={{ newEmail, prevEmail }}
+                            components={[
+                                <span key={0} className='email-confirmation__prevEmail' />,
```

Comment:
To make it bold ,  You can pass   ` components={[<strong key={0} />]}` and avoid adding new class